modules/strap/strap.service.js

```diff
@@ -123,8 +118,7 @@ class StrapService {
 
     await uploadService.deleteFiles(image);
 
-    const uploadImage = await uploadService.uploadSmallImage(image);
-    strap.image = uploadImage.fileNames.small;
+    strap.image = await uploadSmallImage(image);
```

Comment:
In module "uploadService" we don`t have function "uploadSmallImage". We import this function from '../../utils/constructorOptionCommonFilters'